identity_service/db/redis_client.py

```diff
@@ -15,11 +15,12 @@ class RedisClient:
     ----------
     db : redis.Redis
         Redis client
+
     """
 
     db: Redis
```

Comment:
Why not async?
[click](https://redis-py.readthedocs.io/en/stable/examples/asyncio_examples.html)

Reply:
fixed